docker/sandbox-lite/flyte-entrypoint-dind.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+set -euo pipefail
+
+# Apply cgroup v2 hack
+cgroup-v2-hack.sh
+
+trap 'pkill -P $$' EXIT
+FLYTE_TIMEOUT=${FLYTE_TIMEOUT:-600}
+
+monitor() {
+    while : ; do
+        for pid in $@ ; do
+            kill -0 $pid &> /dev/null || exit 1
+        done
+
+        sleep 1
+    done
+}
+
+# Start docker daemon
+echo "Starting Docker daemon..."
```

Comment:
Check `/var/run/docker.pid` if exist then remove it before start

Reply:
i think this is done. cc @evalsocket can you test and ensure it works as expected?